joplin/users/wagtail_hooks.py

```diff
@@ -0,0 +1,34 @@
+import os
+from django.http import HttpResponse
+from django.conf import settings
+from wagtail.admin import messages
+from wagtail.core import hooks
+from smtplib import SMTPException
+from wagtail.admin.forms.auth import PasswordResetForm
+
+
+@hooks.register('after_create_user')
```

Comment:
Using the "after_create_user" hook will run this code within wagtail after making a user through the UI's create user process.